src/components/inputModal/inputModal.tsx

```diff
@@ -0,0 +1,132 @@
+import { Modal, Form } from "react-bootstrap";
+import React, { useEffect } from "react";
+import { useForm, Controller } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import FormButtonGroup from "components/inputModal/FormButton";
+import { schema } from "utils/inputFormSchema";
+import { ModalType } from "utils/enums";
```

Comment:
we prefer all enums with UPPERCASE
```suggestion
import { MODAL_TYPES } from "utils/enums";
```